pulsar-io/elastic-search/pom.xml

```diff
@@ -56,13 +56,12 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <version>3.4</version>
     </dependency>
 
   	<dependency>
   		<groupId>org.elasticsearch.client</groupId>
   		<artifactId>elasticsearch-rest-high-level-client</artifactId>
-  		<version>6.3.2</version>
+  		<version>${elasticsearch.version}</version>
```

Comment:
can you define `elasticsearch-rest-high-level-client` and its version in dependency management? instead of specifying versions here.

Reply:
> can you define `elasticsearch-rest-high-level-client` and its version in dependency management? instead of specifying versions here.

I think just pulsar-io module and tests/integration use this jar, not public.  On the other hand, if we also define  pulsar-io module jar in dependency management, maybe have  dependency conflict.

Reply:
@ambition119 in pulsar, we try to manage all the dependencies in one single place (dependencyManagement at root pom file), so it is easier to find the versions at single place. the only exception we do is when there are version conflicts, we add overwrites when using that jar.